src/containers/search_settings.jsx

```diff
@@ -83,11 +86,48 @@ class SearchSettings extends Component {
       align-items: center;
     `;
 
+    const {
+      FacebookShareButton,
+      EmailShareButton,
+    } = ShareButtons;
+
+    const FacebookShareButtonSetting = {
+      icon: generateShareIcon('facebook'),
+      isRound: true,
+      title: 'GitHub',
+      shareUrl: 'http://github.com',
+      size: 32,
+    };
+
+    const EmailShareButtonSetting = {
+      icon: generateShareIcon('email'),
+      isRound: true,
+      body: 'shared from http://github.com',
+      size: 32,
+      title: 'Sharing SERLER Search Result',
+      shareUrl: 'http://github.com',
+    };
+
```

Comment:
From the description of this feature, it seems we just need to implement either sharing by Facebook or by Email. We do not need to implement both of them.

> Sharing the search results link via Facebook button or email.